### DISCUSSION TITLE
Made a `cast storage` call to a contract on Sepolia Testnet.

### DISCUSSION THREAD

**Discussion by Tanmay53** [Cyfrin/foundry-full-course-cu#545] [Show and tell]
I want to share my first experience with reading the storage data from a contract over the testnet using `cast storage`.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52573153/df2f7a3b-3ca2-4c35-80bd-7581f4719768)

Learnt it from Lesson 7.

Thanks @PatrickAlphaC.

**Comment by theirrationalone** [comment]
Great Observation! 💥💥💥

**Comment by shikhar229169** [comment]
When I run the same command, it is only showing record for storage slot 0, and it then stops. Any idea?

**Reply by Tanmay53** [reply]
Not sure, check the contract address.

**Reply by theirrationalone** [reply]
Might not pointing to contract, verify the the contractName by jumping into broadcast/latest...transactions -> contractName(json encoded)!

**Reply by shikhar229169** [reply]
Oh actually the contract has only 1 variable. Hehe.
I though it will display all slots.

**Reply by theirrationalone** [reply]
> Oh actually the contract has only 1 variable. Hehe.
> I though it will display all slots.

😂😂 we are enough
